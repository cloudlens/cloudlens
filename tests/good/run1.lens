lens l1 (log) {
     stream {
         print("L1 " + e.message)
     }
}

lens l2 (log) {
     stream {
         print("L2 " + e.message)
     }
}

lens l3 (log) {
     stream {
         print("L3 " + e.message)
     }
}

run l1()
run l2()
