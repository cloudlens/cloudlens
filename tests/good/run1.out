L1 foo 1
L2 foo 1
L1 foo 2
L2 foo 2
L1 foo 3
L2 foo 3
L1 foo 4
L2 foo 4
